From: Yue-Lan <lanyue@kylinos.cn>
Date: Wed, 13 Dec 2023 17:14:48 +0800
Subject: update changelog.

---
 peony-qt.pro | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/peony-qt.pro b/peony-qt.pro
index e9b5e9a..3f0cac8 100644
--- a/peony-qt.pro
+++ b/peony-qt.pro
@@ -15,4 +15,4 @@ CONFIG(release,debug|release){
 src.depends = libpeony-qt
 peony-qt-plugin-test.depends = libpeony-qt
 peony-qt-desktop.depends = libpeony-qt
-stable = libpeony-qt
+stable.depends = libpeony-qt
