From: Yue-Lan <lanyue@kylinos.cn>
Date: Wed, 13 Dec 2023 17:39:06 +0800
Subject: update changelog.

---
 stable/ukui3/windows/windows.pro | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/stable/ukui3/windows/windows.pro b/stable/ukui3/windows/windows.pro
index 0925010..ecbbb4f 100644
--- a/stable/ukui3/windows/windows.pro
+++ b/stable/ukui3/windows/windows.pro
@@ -21,7 +21,7 @@ DISTFILES += $$PWD/common.json
 DEFINES += QAPPLICATION_CLASS=QApplication
 
 PKGCONFIG +=gio-2.0 glib-2.0 gio-unix-2.0 gsettings-qt libcanberra libnotify udisks2 openssl dconf
-LIBS +=-L$$PWD/../../../libpeony-qt/ -lgio-2.0 -lglib-2.0 -lX11 -lukui-log4qt
+LIBS +=-L$$PWD/../../../libpeony-qt/ -lgio-2.0 -lglib-2.0 -lX11 -lukui-log4qt -lpeony
 #CONFIG += c++11 link_pkgconfig no_keywords lrelease
 CONFIG += debug link_pkgconfig plugin
 TARGET = stable-main-window
