From: cckylin-cibot <cckylin-cibot@kylinos.cn>
Date: Thu, 9 Jun 2022 03:40:05 +0000
Subject: Fix build errors

---
 libpeony-qt/sound-effect.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libpeony-qt/sound-effect.cpp b/libpeony-qt/sound-effect.cpp
index 9ebb3c5..58ff82d 100644
--- a/libpeony-qt/sound-effect.cpp
+++ b/libpeony-qt/sound-effect.cpp
@@ -67,7 +67,7 @@ QList<char *> SoundEffect::listExistsPath()
 
 void SoundEffect::playAlertSound(QString gsettingStr)
 {
-    gint retval;
+    gint retval = -1;
     const gchar *desc = "Alert Sound";
     QString filenameStr;
     QList<char *> existsPath = this->listExistsPath();
